learn/wavemaker-release-notes/v11-2-0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "WaveMaker 11.2.0 - Release date: 21 November 2022"
+id: ""
+sidebar_label: "v11.2.0 - latest"
+---
+
+New features, such as One-click API Mocking, React Native improvements, and a few bug fixes, are included.
+
+## New Features and Improvements
```

Comment:
New features:
- TrueSPA behaviour. Add a HowTo document. (Anushree)
- Timezone support (Aparna)
- Export swagger from project (Praveen Pokuri)

Bug Fixes:
https://wavemaker.atlassian.net/browse/WMS-22838 (Suchethan)

Reply:
Export Swagger is not explicitly available. So it is not included in this release.